.github/workflows/ci.yml

```diff
@@ -11,29 +11,43 @@ concurrency:
 jobs:
   build-test:
     runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        shell: bash
```

Comment:
I don't see any bash-specific features used.

Reply:
Better that than dash. It's generally better to set what you want and not let GH runner environment decide.

Reply:
IMO this is a very good addition to any GHA workflow.  Keeping to strictly `sh` is not normal behaviour for devs now - and there are some subtle gotchas when not using an explicit `shell` , c.f. https://github.com/rhysd/actionlint/issues/374